components/31-molecules/partner-block/_partner-block.scss

```diff
@@ -23,7 +23,7 @@
 
     li {
       width: calc(50% - #{gutter()});
-      margin: 0 gutter()/2 gutter();
+      margin: 0 gutter()*.5 gutter();
```

Comment:
Add space between * and .5. Makes it clearer that it is a calculation instead of a wildcard.